### ISSUE TITLE
Animation bug by compilig with Angular 5 in production mode on Firefox

### ISSUE THREAD

**Issue by TGotwig** [atomiks/tippyjs#168]
Hey there 🤗
When I compiling my Angular Application with --prod flag,
I have this animation issue on Firefox:

<img src="https://user-images.githubusercontent.com/30773779/35487428-bafda740-047b-11e8-948d-101b773588ae.png">

Without the --prod flag it works perfectly.
I'm using: 
- Angular 5 by Angular CLI.
- Firefox Developer Edition 59.0b4 (64-Bit)

I haven't this Bug on Chrome.

**Comment by atomiks** [comment]
Does this happen on the previous version of FF? (57)

I noticed some weirdness with the latest version as well (although, not that particular bug).

**Comment by TGotwig** [comment]
I don't know.

Because of the --prod flag:
I also noticed that the whole Website crashes,
when one of these three Animations are active:
- shift-toward
- fade
- scale

This is the error message:
TypeError: (void 0) is not a function

It crashes on Firefox and Chrome.

**Comment by TGotwig** [comment]
Hmm the display bug is gone for some reason 😵

But the shift-toward animation looks different,
there are differences between --prod  and without --prod 🤔

Strange bugs 😓

**Comment by atomiks** [comment]
>TypeError: (void 0) is not a function

Which line does it occur on? (Turn off minification for prod for now?)

**Comment by TGotwig** [comment]
<img src="https://user-images.githubusercontent.com/30773779/35489455-69c6bb20-0497-11e8-8121-af98c2e3548a.png"/>

**Comment by atomiks** [comment]
I need to see the line it occurs on in the unminified version.... (particularly how it relates to this library)

**Comment by TGotwig** [comment]
It occurs only at the minified version,
it occurs also very random,
so forget the list I have listed above.

The display bug occurs also randomly.

But the animation "shift-toward" for example,
looks constantly different by using --prod.

**Comment by TGotwig** [comment]
This shows the animation differences of "shift-toward" between production and no-production:
https://mnithmde-my.sharepoint.com/:f:/g/personal/thomas_gotwig_mni_thm_de/Eo7jSe5Z2qpCgrbYh_ieHLQBkR03psvy0YEARx0CpXD4ow?e=ACyMk3

**Comment by atomiks** [comment]
It appears to be transitioning from the top-left of the document in the prod version. But I really have no idea how to fix this since it's so specific to your build

**Comment by TGotwig** [comment]
Yeah right,
I created an small angular project which could reproduce the bug easily,
so it isn't very specific,
you can download it under the url above too,
the tooltip animations for "ng build" and "ng build --prod" are looking different from ground up.

**Comment by atomiks** [comment]
Did you try that option?

**Comment by TGotwig** [comment]
Same as before,
can you reproduce it?

**Comment by atomiks** [comment]
I can't download it right now, can later tonight

**Comment by TGotwig** [comment]
Okay good,
maybe this is an Angular CLI issue,
but I'm not really sure because I don't know what --prod exactly does 😵

**Comment by atomiks** [comment]
I've downloaded your files and can reproduce it when using the `--prod` flag... this is really weird. Is there anything specific about the `--prod` flag that might cause it?

**Comment by atomiks** [comment]
This _should_ be fixed in `v2.1.0`, but I can release a patch if it's still getting removed by the minifier... I couldn't find a way to test to make sure.

```js
this.popper.offsetHeight
```

is now

```js
function reflow(popper) {
  void popper.offsetHeight
}
reflow(this.popper)
```

**Comment by TGotwig** [comment]
Thank you good to hear 🤗🤘
